content/docs/user-guide/contributing/blog.md

```diff
@@ -0,0 +1,139 @@
+# Writing Blog Posts
+
+To create a new post, you need to create a PR to the DVC website
+[project](https://github.com/iterative/dvc.org).
+
+## Prepare your environment
+
+Set up the
+[Development environment](/doc/user-guide/contributing/docs#development-environment)
+to see how your content will look like after you publish it. In the
+`yarn develop` mode you will be able to run the website locally, and it'll be
+updating content in your browser as you edit the post.
+
+## Add a post
+
+Create a Markdown file in the `content/blog` folder. File name must follow the
+pattern: `YYYY-MM-DD-my-new-blog-post.md`. After merging this file into the
+`master` branch, blog post will be published at `blog/my-new-blog-post` address
+automatically.
+
+Write front matter in the following format:
+
+```yml
+---
+title: Hello World
+date: '2015-05-01T22:12:03.284Z'
+description: 'Hello World'
+descriptionLong: |
+  Some long
+  multiline
+  text (supports _basic_ [Markdown](link))
+picture: /uploads/image.jpeg
+pictureComment: Some _Comment_ (supports _basic_ [Markdown](link))
+author: ../authors/author_name.md
+tags:
+  - Open Source
+  - Machine Learning
+  - Data Science
+  - Version Control
+  - AI
+---
+
+```
+
+- `title` - **Required.** Title of the post.
+- `date` - **Required.** Post date. Will be used to sort posts and in RSS.
```

Comment:
What timestamp formats are expected/supported? Can a partial one like `2020-04-12` work?

Reply:
yes, partial is enough (engine will transform it automatically). I'll clarify this.

Reply:
Thanks